botocore/paginate.py

```diff
@@ -186,7 +186,8 @@ def get_paginator(self, operation_name):
 
 class PageIterator:
     """An iterable object to pagiante API results.
-    Please note it is NOT a python iterator"""
+    Please note it is NOT a python iterator.
+    Use ``iter`` to wrap this as a generator"""
```

Comment:
Can we make sure to include a period at the end of the last sentence? We'll also want to make sure the closing `"""` is on its own line.

Reply:
No we can keep it on the same line. The example in the first comment should have what's considered standard formatting for doc strings.

While we don't adhere strictly to PEP8 anymore, the [conventions for docstrings](https://peps.python.org/pep-0008/#documentation-strings) are still widely used. Generally to the effect of:

```python
class MyClass:
    """This is my class.
    
    Extra info beyond the class description.
    """
```

Reply:
Got it thanks! Pushing up now